index.bs

```diff
@@ -24,6 +24,71 @@ Assume Explicit For: yes
 Blah blah blah.
 </section>
 
+<h2 id="deployed-mitigations">
+Some browsers have deployed and announced protections against
+navigational-tracking. This section is a work in progress to detail what
+protections have been shipped and / or are planned. This section should not
+be understood to be comprehensive.
+
+<h3>Safari</h3>
+Safari uses an algorithmic approach to combat navigational-tracking. Safari
+labels a site as having navigational-based cross-site tracking capability if
+the following criteria are met:
```

Comment:
@johnwilander, would it be fair to say

```suggestion
Safari uses an algorithmic approach to combat navigational-tracking. Safari
labels a site as having navigational-based cross-site tracking capability if
the following criteria are met within a particular client:
```

? This vs server-side scanning is an important distinction from the way other browsers are classifying trackers, and while Chrome folks have thrown shade about your choice, it's also important for readers to know that there's prior art for doing it this way.